src/appleseed.shaders/include/appleseed/color/as_color_transforms.h

```diff
@@ -899,9 +900,20 @@ color deltaE_CIEDE2000(
 
     float delta_theta = 30.0 * exp(-sqr((H_bar_prime - 275) / 25));
 
-    float R_T = -2.0 * C_7_sqrt * sin(radians(2 * delta_theta));
+    float C_bar_7 = pow(C_bar, 7);
+
+    float R_T = -2.0 * sqrt(C_bar_7 / (C_bar_7 + pow(25, 7))) *
```

Comment:
You could try to encode it as a floating point value since that's what it really is anyway. No big deal though, was just curious.